app/controllers/settings/exports.js

```diff
@@ -0,0 +1,9 @@
+import Controller from 'ember-controller';
+
+export default Controller.extend({
+  actions: {
+    removeExport(exporter) {
```

Comment:
Can we move this to the route and use `(route-action "...")` but keep the Controller as a created file due the use of `modelTask/taskValue`.